.github/workflows/pull-request-checks.yml

```diff
@@ -89,3 +89,33 @@ jobs:
               run: yarn install --frozen-lockfile
             - name: Run Jest tests
               run: yarn jest
+    lhci:
+        name: Lighthouse
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - name: Use Node.js 16.x
+              uses: actions/setup-node@v1
+              with:
+                  node-version: 16.x
+            - name: Cache node modules
+              uses: actions/cache@v2
+              id: cache
+              with:
+                  path: node_modules
+                  key: ${{ runner.os }}-node-${{ hashFiles('**/yarn.lock') }}
+                  restore-keys: |
+                      ${{ runner.os }}-node-
+            - name: Install dependencies
+              if: steps.cache.outputs.cache-hit != 'true'
+              run: yarn install --frozen-lockfile
+            - name: yarn build
+              run: |
+                  yarn run build
+            - name: Run Lighthouse
+              uses: treosh/lighthouse-ci-action@v7
+              with:
+                  configPath: './lighthouserc.js'
+                  runs: 2
+              env:
+                  LHCI_GITHUB_APP_TOKEN: ${{secrets.LIGHTHOUSE_CI_TOKEN}}
```

Comment:
what is this token ?

Reply:
For the lighthouse ci app. Though I will need to revise this bc it isn't working as it should. Like you see here https://github.com/apps/lighthouse-ci We should see the preview like so:

![image](https://user-images.githubusercontent.com/6556242/150434505-44e04bd1-6d8a-41f3-904e-b908b7d811b2.png)

Reply:
I will take a look first thing tomorrow @harikakotipalli